package.json

```diff
@@ -171,6 +178,7 @@
     "p-queue": "^8.0.1",
     "p-ratelimit": "^1.0.1",
     "pretty-ms": "7.0.1",
+    "process": "^0.11.10",
```

Comment:
I don't believe this will be needed with polyfill plugin

Reply:
See https://github.com/shapeshift/web/pull/8924#discussion_r1982944416